AsyncDisplayKit/ASPagerNode.m

```diff
@@ -84,6 +84,17 @@ - (ASCellNode *)collectionView:(ASCollectionView *)collectionView nodeForItemAtI
   return pageNode;
 }
 
+- (ASDataControllerCellNodeBlock)collectionView:(ASCollectionView *)collectionView nodeBlockAtIndexPath:(NSIndexPath *)indexPath {
+  ASDisplayNodeAssert(_pagerDataSource != nil, @"ASPagerNode must have a data source to load nodes to display");
+  if (![_pagerDataSource respondsToSelector:@selector(pagerNode:nodeBlockAtIndex:)]) {
+    ASCellNode *node = [_pagerDataSource pagerNode:self nodeAtIndex:indexPath.item];
+    return ^{ return node; };
+  }
+  ASDataControllerCellNodeBlock block = [_pagerDataSource pagerNode:self nodeBlockAtIndex:indexPath.item];
+  ASDisplayNodeAssertNotNil(block, @"Invalid node block. Block should be non-nil.");
```

Comment:
Presumably this is asserted at the level of the caller of this method, so let's eliminate two lines in just directly return the data source call.